oauth2client/contrib/gce.py

```diff
@@ -51,13 +59,17 @@ class AppAssertionCredentials(AssertionCredentials):
     """
 
     @util.positional(2)
```

Comment:
OK. The decorator only has [an opionion][1] about the **max** number of positional, so we are good to make `scope` optional?

[1]: https://github.com/google/oauth2client/blob/0c7996099f9e3d74f7b9a6d32b6f69f44987638a/oauth2client/util.py#L48